src/main/java/org/team1540/robot2019/Robot.java

```diff
@@ -49,6 +67,18 @@ public void robotInit() {
         hatch = new HatchMech();
         climber = new Climber();
 
+        odometry = new TankDriveOdometryRunnable(
+            drivetrain::getLeftPositionMeters,
+            drivetrain::getRightPositionMeters,
+            Hardware.navx::getAngleRadians,
+            0.011
+        );
+
+        limelight = new Limelight("limelight-a");
+        limelightLocalization = new LimelightLocalization(limelight, 0.05); // Doesn't have to be very frequent if things that use it also call update
+
+        tebPlanner = new TEBPlanner(() -> new Odometry(odometry.getOdomToBaseLink(), drivetrain.getTwist()), 5801, 5800, "10.15.40.202", 0.01);
```

Comment:
Wait are we still using TEB? I was under the impression that that wasn't happening. If not, then this should obviously be removed

Reply:
I still think we should try to use it for profiling during auto. We can use pure pursuit/Pathfinder as a backup.

Reply:
Ok